.github/workflows/haskell.yml

```diff
@@ -115,7 +115,10 @@ jobs:
         TMPDIR: ${{ runner.temp }}
         TMP: ${{ runner.temp }}
         KEEP_WORKSPACE: 1
-      run: cabal test all
+      run: |
+        cabal test all
+        rm -rf cardano-cli/test/cardano-cli-golden/files/golden
```

Comment:
I think this `rm` followed by rerunning the test suite should be explained by a comment.

It looks like an unusual pattern and we will forget why it's there in the future.

Reply:
Oh woow yeah I overlooked that. Windows is so unusual, my mind doesn't even consider it :laughing: